lib/cartopy/tests/crs/test_equidistant_conic.py

```diff
@@ -23,12 +24,20 @@ def test_default(self):
                       'y_0=0.0', 'lat_1=20.0', 'lat_2=50.0'}
         check_proj_params('eqdc', eqdc, other_args)
 
-        assert_almost_equal(np.array(eqdc.x_limits),
-                            (-22784919.35600352, 22784919.35600352),
-                            decimal=7)
-        assert_almost_equal(np.array(eqdc.y_limits),
-                            (-10001965.729313632, 17558791.85156368),
-                            decimal=7)
+        if pyproj.__proj_version__ >= '9.2.0':
```

Comment:
What are your thoughts about changing the number of decimal places we compare to here? It seems like if we can compare to 2-3+ decimal places that should be good enough for Cartopy's purposes. We are really relying on proj for all the "official" transformations.

Reply:
I don't have any strong opinion about number of decimals. I do find it strange that cartopy supports different versions of proj that would lead to very different results (eg boundary of UK) 

I would either not test values from proj at all, since they change between versions OR only support one version of proj that is considered "correct", most likely latest version of proj.

that's just my 2 cents but no need to change anything for this PR

Reply:
I agree with everything you say here :)